packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -0,0 +1,62 @@
+/*
+  All the utility functions and configs related to our testing
+*/
+
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+
+export const l1NetworkConfig = 'goerli'
+
+export const l2NetworkConfig = {
+  networkName: 'arbitrum goerli',
+  rpcUrl: 'https://goerli-rollup.arbitrum.io/rpc',
+  chainId: '421613',
+  symbol: 'ETH',
+  blockExplorer: 'https://goerli-rollup-explorer.arbitrum.io',
+  isTestnet: true
+}
+
+export const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env(
+  'INFURA_KEY'
+)}`
+export const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
+
+export async function getInitialETHBalance(rpcURL: string): Promise<BigNumber> {
+  const goerliProvider = new JsonRpcProvider(rpcURL)
+  return await goerliProvider.getBalance(Cypress.env('ADDRESS'))
+}
+
+export const setupMetamask = (isL2Network: boolean) => {
+  // we want complete control over the metamask flow before our web app starts (because we might want to start from an L2 network)
+  // hence we are not bootstrapping metamask from.env anymore
+
+  // start with bootstrapping from default l1 network
+  return cy
+    .setupMetamask(
+      Cypress.env('SECRET_WORDS'),
+      l1NetworkConfig,
+      Cypress.env('METAMASK_PASSWORD')
```

Comment:
I think we can remove the env var and just use a string like `THIS_IS_THE_METAMASK_PASSWORD` because synpress will set up the in-test metamask extension with this custom string as pw and reuse it within the test run but it has nothing to do with the metamask password we normally use on our browser
so we don't need user input for this

what do you think?